src/index.ts

```diff
@@ -19,7 +23,7 @@ const defaultApps = [
   require('./plugins/default')
 ]
 
-module.exports = (options = {}) => {
+export const createProbot = (options: Options) => {
```

Comment:
it would be very nice to have an interface as the returned type, so that we can get autocompletion for the different functions
I suggest:
```ts
export interface Probot {
  server: Server;
  webhook: Webhooks;
  receive: (event: WebhookEvent) => Promise<any>[];
  logger: Logger;
  load: (plugin: string | Plugin) => Robot;
  setup: (apps: Array<string | Plugin>) => void;
  start: () => void;
}